app/models/post.rb

```diff
@@ -0,0 +1,20 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
+
+  validates :title, presence: true
+  validates :text, presence: true
+
+  after_save :update_user_posts_counter
+
+  private
+
+  def update_user_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
+
+  def return_recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] This method should not be private.  Making it private means that one cannot call the method upon a Post object like this `post.return_recent_comments` which we don't want. We want to be able to call the method on post objects. Kindly remove it under the private method grouping.

  ![image](https://github.com/bhushan354/Rails-Blog-App/assets/81468675/77b5cda7-46bf-4c8a-8913-c01f234e9c21)